common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -229,10 +229,13 @@ public void set(String key, String value) {
   }
 
   public String get(String key, final String defaultValue) {
-    String raw = mProperties.getProperty(key, defaultValue);
-    String updated = lookup(raw);
-    LOG.debug("Get Tachyon property {} as {} with default {}", key, updated, defaultValue);
-    return updated;
+    if (mProperties.containsKey(key)) {
```

Comment:
This method will be gone very soon. 
Could you not rely on using this method but `public String get(String key)` ?